sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -212,6 +212,51 @@ public TableClient(string connectionString, string tableName, TableClientOptions
             Name = tableName;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TableClient"/> using the specified <see cref="Uri" /> and <see cref="TokenCredential"/>.
+        /// </summary>
+        /// <param name="endpoint">
+        /// A <see cref="Uri"/> referencing the table service account.
+        /// This is likely to be similar to "https://{account_name}.table.core.windows.net/{table_name}"
+        /// or "https://{account_name}.table.cosmos.azure.com/{table_name}".
+        /// </param>
+        /// <param name="tableName">The name of the table with which this client instance will interact.</param>
+        /// <param name="tokenCredential">The <see cref="TokenCredential"/> used to authorize requests.</param>
+        /// <param name="options">
+        /// Optional client options that define the transport pipeline policies for authentication, retries, etc., that are applied to every request.
+        /// </param>
+        /// <exception cref="ArgumentException"><paramref name="endpoint"/> is not https.</exception>
```

Comment:
nit:  `not https` sounds awkward to me.   However, `not secure` doesn't seem to be explicit enough to help.   I don't have a better suggestion, but thought it worth mentioning for consideration.

Reply:
"... is not a TLS Uri" ?

Reply:
That reads well to me.   I wonder if it's obvious enough to developers troubleshooting, though.

Reply:
"...Uri should begin with 'https'" ?

Reply:
That, I really like!